index.json

```diff
@@ -72,5 +72,13 @@
             "categories-2"
         ],
         "url": "tags/tournament/modes2.md"
+    }, {
+        "identifier": "pvp-clients",
+        "aliases": [
+            "lunar",
+            "badlion",
+            "optifine"
```

Comment:
optifine is not a pvp client; thats its own separate issue

Reply:
Updated it, removed optifine from aliases